ibm_mq/datadog_checks/ibm_mq/connection.py

```diff
@@ -56,9 +56,20 @@ def get_ssl_connection(config):
     sco = pymqi.SCO()
     sco.KeyRepository = pymqi.ensure_bytes(config.ssl_key_repository_location)
 
-    queue_manager = pymqi.QueueManager(None)
-    queue_manager.connect_with_options(config.queue_manager_name, cd, sco)
+    if config.ssl_certificate_label:
+        sco.CertificateLabel = pymqi.ensure_bytes(config.ssl_certificate_label)
+
+    connect_options = {}
+    if config.username and config.password:
```

Comment:
We could add debug log lines similar to what `get_normal_connection` does for consistency.

Reply:
Added this instead, can be very helpful for support. 

```

    log.debug(
        "Create SSL connection with ConnectionName=%s, ChannelName=%s, Version=%s, SSLCipherSpec=%s, "
        "KeyRepository=%s, CertificateLabel=%s, user=%s",
        cd.ConnectionName,
        cd.ChannelName,
        cd.Version,
        cd.SSLCipherSpec,
        sco.KeyRepository,
        sco.CertificateLabel,
        connect_options.get('user'),
    )
```

Example:
```
2020-09-11 09:03:47 UTC | CORE | DEBUG | (pkg/collector/python/datadog_agent.go:123 in LogMessage) | - | (connection.py:71) | Create SSL connection with ConnectionName=b'localhost(11414)', ChannelName=b'PYMQI.SSL.SVRCONN', Version=6, SSLCipherSpec=b'TLS_RSA_WITH_AES_256_CBC_SHA256', KeyRepository=b'/opt/pki/keys/client', CertificateLabel=b'client', user=admin
```